www-apps/sonarr/sonarr-3.0.8.1507.ebuild

```
@@ -0,0 +1,42 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+SRC_URI="https://download.sonarr.tv/v3/main/${PV}/Sonarr.main.${PV}.linux.tar.gz"
+
+DESCRIPTION="Sonarr is a Smart PVR for newsgroup and bittorrent users."
+HOMEPAGE="https://www.sonarr.tv"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
```

```suggestion
KEYWORDS="-* ~amd64"
```

(I think it's a binary?)

---

Sonarr is (currently) a CLR executable, so it should be runnable on any arch that has mono.